lib/node_modules/@stdlib/blas/ext/base/dsum/manifest.json

```
@@ -40,35 +40,41 @@
         "@stdlib/napi/export",
         "@stdlib/napi/argv",
         "@stdlib/napi/argv-int64",
-        "@stdlib/napi/argv-strided-float64array"
+        "@stdlib/napi/argv-strided-float64array",
```

```suggestion
        "@stdlib/napi/argv-strided-float64array",
        "@stdlib/napi/create-double",
```

I believe you are missing a dep.